00_basics/sc_dimensions.ipynb

```diff
@@ -0,0 +1,771 @@
+{
```

Comment:
<p>The best version I have found so far is:</p><pre class="ql-syntax" spellcheck="false">~matrixMul = { |a, b|
&nbsp;&nbsp;&nbsp; if(a[0].size != b.size) { Error("matrix sizes do not match").throw };
    a.collect { |d| (b * d).sum }
};
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/capital-G/musikinformatik-sose2021/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>